app/utilities/commands.py

```
@@ -0,0 +1,199 @@
+
+        imagem_dados_paciente = Image.new("RGB", (1200, 308), color=(255, 255, 255))
+        draw_img = ImageDraw.Draw(imagem_dados_paciente)
+
+        font1 = ImageFont.truetype("./Fontes/verdanab.ttf", 55)
+        font2 = ImageFont.truetype("./Fontes/verdana.ttf", 55)
+
+        nome_paciente_escrever = info_dict["patient_datum"]["name"]
+        nome_dr = info_dict["dentist_datum"]["name"]
+        nome_dr_escrever = f"Dr(a). {nome_dr}"
+
+        birth_date = datetime.datetime.strptime(
+            info_dict["patient_datum"]["birthdate"], "%Y-%m-%d"
+        )
+        end_date = datetime.datetime.strptime(
+            info_dict["date"][:-10], "%Y-%m-%dT%H:%M:%S"
+        )
+        idade = (end_date - birth_date) / datetime.timedelta(days=365.2425)  # 365.2425
+        idade_escrever = f"{int(idade)}a {int((idade - int(idade)) * 12)}m"
+        nascimento = birth_date.strftime("%d/%m/%Y")
+        nascimento_escrever = f"Data Nasc.: {nascimento} Idade: {idade_escrever}"
+        data = end_date.strftime("%d/%m/%Y")
+        data_escrever = f"Data: {data}"
+
+        left_border_dist = 35
+        draw_img.text(
+            (left_border_dist, 35), nome_paciente_escrever, font=font1, fill=(0, 0, 0)
+        )
+        draw_img.text(
+            (left_border_dist, 105), nome_dr_escrever, font=font2, fill=(0, 0, 0)
+        )
+        draw_img.text(
+            (left_border_dist, 175), nascimento_escrever, font=font2, fill=(0, 0, 0)
+        )
+        draw_img.text(
+            (left_border_dist, 245), data_escrever, font=font2, fill=(0, 0, 0)
+        )
+
+        imagem_dados_paciente = np.array(imagem_dados_paciente)
+        return imagem_dados_paciente, nome_paciente_escrever
+
+    # print('''Carregando imagens...\nAguarde alguns instantes...''')
+    try:
+        patient_id = int(patient_id)
+    except ValueError:
+        try:
+            patient_id = int(patient_id.split("-")[0])
+        except ValueError:
+            patient_id = input(
```

It should be a pop up, not an "input". As an initial solution, one could just let the error be thrown and let the top level handle the error/pop up later.